src/s1ap/handlers/s1setup.c

```diff
@@ -133,13 +136,50 @@ create_s1setup_response(/*enb info,*/unsigned char **s1_setup_resp)
 	return resp_buf.pos;
 }
 
+/**
+ * @brief get socket info and add assoc info to hash
+ * @return error code.
+ */
+int
+addEnbInfoToHash(int enbfd, int inStreamId)
+{
+    struct sctp_status stat = {0};
+    struct enb_assoc_info enbAssoc = {0};
+    int stat_len = sizeof(struct sctp_status);
+    memset (&stat, 0, sizeof (struct sctp_status));
+    memset (&enbAssoc, 0, sizeof (struct enb_assoc_info));
```

Comment:
stat & enbAssoc already initialized at the time of variable declaration using {0}

Reply:
fixed